specification/hybridcloud/resource-manager/readme.md

```diff
@@ -0,0 +1,79 @@
+# Hybrid Cloud API
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Hybrid Cloud API.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Hybrid Cloud API.
+
+```yaml
+openapi-type: arm
+tag: package-2023-01-01-preview
+title: HybridCloudClient
+```
+
+### Tag: package-2023-01-01-preview
+
+These settings apply only when `--tag=package-2023-01-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-01-01-preview'
+input-file:
+  - Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
+  - repo: azure-resource-manager-schemas
+  - repo: azure-cli-extensions
+  - repo: azure-powershell
```

Comment:
Could we add a line to include `azure-sdk-for-net-track2` as well?

Reply:
I did not see the `azure-sdk-for-net-track2` entry in this list. Could you double check?